gqcp/include/Operator/SecondQuantized/MixedUSQTwoElectronOperatorComponent.hpp

```diff
@@ -127,15 +127,15 @@ class MixedUSQTwoElectronOperatorComponent:
         for (size_t i = 0; i < this->numberOfComponents(); i++) {
             switch (sigma) {
             case Spin::alpha: {
-                const auto temp = T_conjugate.template einsum<1>("UQ,TUVW->TQVW", parameters[i]);
-                const auto transformed = T_tensor.template einsum<1>("TP,TQVW->PQVW", temp);
+                const auto temp = T_tensor.template einsum<1>("UQ,TUVW->TQVW", parameters[i]);
+                const auto transformed = T_conjugate.template einsum<1>("TP,TQVW->PQVW", temp);
```

Comment:
Good catch on the fact that the indices which have to be transformed with the conjugate matrix were switched here!

Reply:
Yeah, I was re-checking the indices and I noticed something was off... :)